### PR TITLE
Added Kruskal and Dijkstra Algorithm in C++

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
Fixes a part of #7

### CODE REVIEW

File: Code/C++/Kruskal.cpp [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+
+Following is The Core Implementation of Kruskal Algorithm
+*/
+#include <iostream>
+#include <algorithm>
+#include <vector>
+using namespace std;
+
+struct edge
```
Comment (rudrakshi99) [inline Code/C++/Kruskal.cpp:15]: use **class** instead of **struct** and declare class members under public.

**Comment by rudrakshi99** [comment]
@Siddhant-K-code Add comments to the new line just above your code with **4 tabs** indentation. And also add at least **2 test cases** in your code.

File: Code/C++/Dijkstra.cpp [hunk 2]
```diff
@@ -23,12 +23,17 @@ typedef vector<int> vi;
 typedef pair<int, int> pii;
 typedef vector<pii> vii;
 typedef pair<int, int> pii;
-vii *Graph;  // Graph -> Vector - Pair
-vi Distance; // It Stores the Distance of every other node from start.
+
+// Graph -> Vector - Pair
```
Comment (rudrakshi99) [inline Code/C++/Dijkstra.cpp:27]: use **4 tabs**  to indent the comment.

File: Code/C++/Dijkstra.cpp [hunk 3]
```diff
@@ -23,12 +23,17 @@ typedef vector<int> vi;
...
+vii *Graph;
+
+// It Stores the Distance of every other node from start.
```
Comment (rudrakshi99) [inline Code/C++/Dijkstra.cpp:30]: use **4 tabs**  to indent the comment.

File: Code/C++/Dijkstra.cpp [hunk 4]
```diff
@@ -23,12 +23,17 @@ typedef vector<int> vi;
...
+vi Distance;
 
 void Dijkstra(int start, int N)
 {
-    priority_queue<pii, vector<pii>, greater<pii>> Q; // min heap
+    // min heap
```
Comment (rudrakshi99) [inline Code/C++/Dijkstra.cpp:35]: use **4 tabs**  to indent the comment.

Comment (Siddhant-K-code) [reply Code/C++/Dijkstra.cpp:35]: isn't this one is 4 tabs indent?

Comment (rudrakshi99) [reply Code/C++/Dijkstra.cpp:35]: No, I have checked it in VS Code and it's 1 tab indent. And you also need to indent all the inline comments in the main section of the Dijkstra Algorithm by 4 tabs.

Comment (rudrakshi99) [reply Code/C++/Dijkstra.cpp:35]: Take the reference for indentation from [this code.](https://github.com/Algo-Phantoms/Algo-Tree/blob/main/Code/C%2B%2B/BFS.cpp)

Comment (Siddhant-K-code) [reply Code/C++/Dijkstra.cpp:35]: I am using Beautifier/prettier. So, No Issue !!

**Comment by Siddhant-K-code** [comment]
Please Review it.

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**